tests/worker_deployment_test.go

```diff
@@ -40,15 +41,15 @@ type (
 
 func TestWorkerDeploymentSuite(t *testing.T) {
 	t.Parallel()
-	t.Run("sync", func(t *testing.T) {
-		suite.Run(t, &WorkerDeploymentSuite{workflowVersion: workerdeployment.InitialVersion})
-	})
-	t.Run("async", func(t *testing.T) {
-		suite.Run(t, &WorkerDeploymentSuite{workflowVersion: workerdeployment.AsyncSetCurrentAndRamping})
-	})
-	t.Run("version_rev_no", func(t *testing.T) {
-		suite.Run(t, &WorkerDeploymentSuite{workflowVersion: workerdeployment.VersionDataRevisionNumber})
-	})
+	//t.Run("sync", func(t *testing.T) {
+	//	suite.Run(t, &WorkerDeploymentSuite{workflowVersion: workerdeployment.InitialVersion})
+	//})
+	//t.Run("async", func(t *testing.T) {
+	//	suite.Run(t, &WorkerDeploymentSuite{workflowVersion: workerdeployment.AsyncSetCurrentAndRamping})
+	//})
+	//t.Run("version_rev_no", func(t *testing.T) {
+	suite.Run(t, &WorkerDeploymentSuite{workflowVersion: workerdeployment.VersionDataRevisionNumber})
```

Comment:
Maybe we should use [this pattern](https://github.com/temporalio/temporal/blob/65044e156a6f231159800eeb97f2b141a3f41e7f/service/matching/matching_engine_test.go#L144) so we don't have to comment/uncomment stuff